### ISSUE TITLE
Investigate how DS team can be notified and view community space submissions

### ISSUE THREAD

**Issue by robertjmccarthy** [ministryofjustice/moj-frontend#1043]
## What
Investigate how the MoJ Design System team can be notified that a new component has been submitted to the community space, and how the team can view the submission and related assets. This investigation is to include how the team can be notified, and view, that an addition has been submitted to an existing component in the community space.

Submissions to the community space will be through a form on the MoJ Design System documentation website.

Share findings and suggestions with the team working on the community space.

## Why
When a component is to be submitted to the community space, the MoJ Design System team will need to (a) review the submission, and (b) upload the submission to the community space on the MoJ Design System documentation site. To do both of these tasks, the MoJ Design System needs to be notified when a submission has been made and be able to view the submission and related assets.

## Who needs to be involved
- Developer (leading)
- Interaction Designer (supporting)
- Content Designer (supporting)

## Done
When findings and suggestions are shared with the team working on the community space to inform the work to develop a contribution form.

**Comment by jbarnardmoj** [comment]
I looked into using notify for sending the team email when a submission has been made. We will want to add a URL or link to a file to enable the team to see the details of the submission via the email.

For example a link to a page or (perhaps simpler) a link to a file with the submission details.

**Comment by jbarnardmoj** [comment]
Notify email allows an attached file to be a maximum of 2mb and has a restriction on file type (so no zip files and certainly no code extensions).

This leads us to likely want to include a link to a downloadable file rather than send an attached file.

We could store the submitted assets in a database or S3 and allow a download.

S3 would be a simpler option for storage and retrieval could be direct or indirect via our backend.

We would have to consider how we could control access to downloadable assets.

**Comment by mattnixon1** [comment]
Can we make use of the microsoft suite and place things in the onedrive. I'm assuming s3 is way easier to implement but the access is trickier

**Comment by robertjmccarthy** [comment]
Could we make use of Github? Would be good to chat about this point in this afternoon's catch-up.

**Comment by jbarnardmoj** [comment]
Yes we can use either; github is probably easier to get up and running and potentially a better option as we know we all have access. 

We would use the github API to make a commit. Perhaps we would create a new branch on a code repo in github and add submitted files (including the form submission which could include generated markdown files) to a new branch. That way we would have a branch that represents the submission.

The only potentially issue could be a limit on the size of files being added - you wouldn't expect a large file to be added to a code repository. But I think we're only talking about screenshots, code snippets and form submissions / generated markdown files.

**Comment by jbarnardmoj** [comment]
Moving to done; creating a PR in github will trigger a notification to the team.
